sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/flow/FlowRuleManager.java

```diff
@@ -129,20 +130,16 @@ public static boolean isOtherOrigin(String origin, String resourceName) {
         @Override
         public void configUpdate(List<FlowRule> value) {
             Map<String, List<FlowRule>> rules = FlowRuleUtil.buildFlowRuleMap(value);
-            if (rules != null) {
-                flowRules.clear();
-                flowRules.putAll(rules);
-            }
+            //the rules was always not null, it's no need to check nullable
+            //remove checking to avoid IDE warning
+            flowRules.set(rules);
             RecordLog.info("[FlowRuleManager] Flow rules received: {}", flowRules);
```

Comment:
Maybe the variable in logging message should be updated now.

Reply:
fixed